addons/sale_project/static/tests/tours/project_create_sol_tour.js

```diff
@@ -0,0 +1,98 @@
+import { markup } from "@odoo/owl";
+import { registry } from "@web/core/registry";
+import { stepUtils } from "@web_tour/tour_service/tour_utils";
+import { _t } from "@web/core/l10n/translation";
+
+registry.category("web_tour.tours").add('project_create_sol_tour', {
+    test: true,
+    url: "/web",
+    rainbowManMessage: _t("Congratulations! You have successfully created a sales order item on the fly from a Project."),
+    steps: () => [stepUtils.showAppsMenuItem(),
+{
+    trigger: ".o_app[data-menu-xmlid='project.menu_main_pm']",
+    position: "bottom",
+}, {
+    trigger: ".o-kanban-button-new",
+    extra_trigger: ".o_project_kanban",
+    content: markup(_t("Let\'s create your first <b>project</b>.")),
+    position: "bottom",
+    width: 200,
+}, {
+    trigger: ".o_project_name input",
+    content: markup(_t("Choose a <b>name</b> for your project. <i>It can be anything you want: the name of a customer, of a product, of a team, of a construction site, etc.</i>")),
+    position: "right",
+    run: "text DIY Room Decor",
+}, {
+    trigger: ".o_form_label:contains('Billable')",
+    extra_trigger: ".o_project_kanban",
+    content: _t("Make project Billable"),
+    position: "bottom",
+}, {
+    trigger: ".o_open_tasks",
+    content: markup(_t("Let\'s create your first <b>project</b>.")),
+    position: "bottom",
+}, {
+    trigger: ".o_breadcrumb .o_back_button",
+    position: "right",
+}, {
+    trigger: ".o_kanban_record:contains('DIY Room Decor'):first .o_dropdown_kanban .dropdown-toggle",
+    position: "bottom",
+}, {
+    trigger: ".o_kanban_card_manage_settings a:contains('Settings')",
+    content: _t("Open team form view"),
+    position: "bottom",
+}, {
+    trigger: ".o_field_widget[name='partner_id']",
+    position: "bottom",
+    run() {
+        document.querySelector(".o_field_widget[name='partner_id'] input").click();
+    }
+}, {
+    trigger: ".ui-autocomplete > li > a:not(:has(i.fa))",
+    content: _t("Select customer"),
+    position: "bottom",
+    auto: true,
+}, {
+    id: "project_sale_timesheet_start",
+    trigger: 'a.nav-link[name="settings"]',
+    extra_trigger: 'div.o_notebook_headers',
+    content: 'Click on Settings page to check the allow_billable checkbox',
+}, {
+    trigger: ".o_field_widget[name='sale_line_id']",
+    content: _t("Create Sale order line on fly"),
+    position: "bottom",
+    run() {
+        document.querySelector(".o_field_widget[name='sale_line_id'] input").click();
+    }
```

Comment:
Is it not equivalent to:
```suggestion
    trigger: ".o_field_widget[name='sale_line_id'] input",
    content: _t("Create Sale order line on fly"),
    position: "bottom",
```